CMakeLists.txt

```diff
@@ -2,101 +2,35 @@ cmake_minimum_required(VERSION 2.6)
 
 project(FreeGish C)
 
-set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake-extras/;${CMAKE_MODULE_PATH}")
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake-extras)
 
 option(INSTALL_FHS "Install into Linux FHS" OFF)
+set(OpenGL_GL_PREFERENCE LEGACY)
 
-#don't pollute source dir
-if(NOT ${INSTALL_FHS})
-    set(EXECUTABLE_OUTPUT_PATH "${PROJECT_SOURCE_DIR}")
-endif(NOT ${INSTALL_FHS})
+include(FindPkgConfig)
```

Comment:
This can be removed. pkg-config is not used anymore in the CMake.

Reply:
Ah that's new. Nice! I tried removing pkg-config from my environment and it still works :+1: I removed it from the readme as well